app/Repositories/UserRepository.php

```diff
@@ -32,4 +32,11 @@ public function find($user_id)
 
         return $userEntity;
     }
+
+    public function followingTweets($user_id)
+    {
+        $user = $this->user->where('id', $user_id)->first();
+
+        return $user->following()->with('tweets')->get();
+    }
```

Comment:
you should not get the tweets from the user repository. instead, you can simply return the ids of the users you want.
```php
    public function followingTweets($user_id)
    {
        return $this->user->find($id)->following()->pluck('id');
    }
```